openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -131,39 +134,48 @@ All OAuth 2.0 Grant Types and extensions mechanisms can be used in conjunction w
 
-Note: "type" and "doctype" are terms defined by an individual Credential format. For details, see (#format_profiles).
+- Same Credential Dataset and Format, but different keys
+- Different Credential Dataset and Format, but same key
+- Different Credential Dataset and Format and different keys
 
-The Wallet MAY send one Batch Credential Request to the Batch Credential Endpoint to request the following in the Batch Credential Response:
+The Credential Format represents the Verifiable Credential and defines how the data (or Dataset) within the Credential is organized. This can include the specific parameters needed to describe the Credential, which are established in the Credential Format Profile. Therefore, the Dataset and the Credential Format are closely connected. The Dataset provides the actual data or claims, while the Credential Format provides the structure or schema for representing and interacting with this data.
 
-* multiple Credentials of different types/doctypes bound to the same proof, or 
-* multiple Credentials of the same type/doctype bound to different proofs, or
-* multiple Credentials of different types/doctypes bound to different proofs.
+This specification allows for the issuance of Verifiable Credentials through two types of endpoints: the Single Credential Endpoint and the Batch Credential Endpoint.
 
-In the course of the authorization process, the Credential Issuer MAY also request Credential presentation as a means to authenticate or identify the End-User during the issuance flow, as illustrated in (#use-case-5).
+1. Single Credential Endpoint: This endpoint is used when a single Verifiable Credential is to be issued. The request to this endpoint includes the necessary parameters for the issuance of one credential. The response from this endpoint will contain the issued Verifiable Credential.
+
+2. Batch Credential Endpoint: This endpoint is used when multiple Verifiable Credentials need to be issued at once. The request to this endpoint includes the necessary parameters for the issuance of multiple credentials. The response from this endpoint will contain all the issued Verifiable Credentials.
 
-At its core, this specification is Credential format agnostic and allows implementers to leverage specific capabilities of Credential formats of their choice. Multiple Credential formats can be used within the same transaction. 
+When dealing with multiple Credentials, the following combinations are possible:
 
-The specification achieves this by defining the following:
+1. Multiple Credentials with different Credential Format bound to the same Cryptographic Holder Binding.
```

Comment:
this does not accommodate for issuing multiple credentials with different Datasets

Reply:
I rewrote this section, removing this enumeration.